drivers/npmx/npmx_shell.c

```diff
@@ -3008,6 +3008,8 @@ SHELL_STATIC_SUBCMD_SET_CREATE(sub_buck_gpio,
 
 /* Creating subcommands (level 4 command) array for command "buck status power". */
 SHELL_STATIC_SUBCMD_SET_CREATE(sub_buck_status_power,
+			       SHELL_CMD(set, NULL, "Set buck power status",
+					 cmd_buck_status_power_set),
```

Comment:
`set` command should be after `get` command

Reply:
fixed